### PR TITLE
Phisical to Physical

### PR DESCRIPTION
**Pull request by UnknownMurders**
Incorrect spelling

### CODE REVIEW

**Comment by UnknownMurders** [comment]
!review

File: vueui/src/components/view/records/general.vue [hunk 1]
```diff
@@ -10,7 +10,7 @@
       <vui-group-item label="Age:"><view-records-field :editable="(editable & 1) > 0" path="active.age"/></vui-group-item>
       <vui-group-item label="Sex:"><view-records-field :editable="(editable & 1) > 0" path="active.sex"/></vui-group-item>
       <vui-group-item label="Rank:"><view-records-field :editable="(editable & 1) > 0" path="active.rank"/></vui-group-item>
-      <vui-group-item label="Phisical Status:">
+      <vui-group-item label="Physical Status:">
         <view-records-field :editable="(editable & 1) > 0" path="active.phisical_status">
           <select v-model="$root.$data.state.editingvalue">
             <option v-for="i in choices.phisical_status" :key="i" :value="i">{{ i }}</option>
```
Comment (Karolis2011) [inline vueui/src/components/view/records/general.vue]: Well, maybe fix actually variable name?

Comment (UnknownMurders) [reply vueui/src/components/view/records/general.vue]: Well, I wouldn't want to screw the whole thing up but sure. I can fix the entire active.phisical_status. Lemme take a look.

File: vueui/src/components/view/records/general.vue [hunk 2]
```diff
@@ -10,10 +10,10 @@
       <vui-group-item label="Age:"><view-records-field :editable="(editable & 1) > 0" path="active.age"/></vui-group-item>
       <vui-group-item label="Sex:"><view-records-field :editable="(editable & 1) > 0" path="active.sex"/></vui-group-item>
       <vui-group-item label="Rank:"><view-records-field :editable="(editable & 1) > 0" path="active.rank"/></vui-group-item>
-      <vui-group-item label="Phisical Status:">
-        <view-records-field :editable="(editable & 1) > 0" path="active.phisical_status">
+      <vui-group-item label="Physical Status:">
+        <view-records-field :editable="(editable & 1) > 0" path="active.physical_status">
           <select v-model="$root.$data.state.editingvalue">
-            <option v-for="i in choices.phisical_status" :key="i" :value="i">{{ i }}</option>
+            <option v-for="i in choices.phsyical_status" :key="i" :value="i">{{ i }}</option>
```
Comment (Kaedwuff) [inline vueui/src/components/view/records/general.vue]: You implemented a different typo instead

Comment (Sindorman) [reply vueui/src/components/view/records/general.vue]: Yup

**Pull request closed without merging**